### ISSUE TITLE
Feature request: cluster and latent class analysis

### ISSUE THREAD

**Issue by JamesChrHarman** [jasp-stats/jasp-desktop#1935]
## Environment (for enhancement requests)
 
* Enhancement: Inclusion of two-step cluster and latent class analysis techniques
* Purpose: This would help analysts to estimate latent nominal classes in addition to the latent interval factors approximated by factor analysis.

**Comment by FransMeerhoff** [comment]
Hi @EJWagenmakers

Can you have a look at this request

**Comment by AlexanderLyNL** [comment]
This is definitely an interesting enhancement, though, I think that we should implement some simpler stuff first. I'll plan this into one of our later releases.

**Comment by EJWagenmakers** [comment]
I agree with Alexander. Very interesting enhancement, also from a Bayesian perspective, but for a later release.

**Comment by JamesChrHarman** [comment]
Thank you! Perhaps my request for data weighting could be considered as a higher priority, though? It's pretty essential when working with survey data.
